### PR TITLE
10 hfc410a

### PR DESCRIPTION
**Pull request by znicholls**
Add a unit context to make splitting HFC410a into its parts slightly easier

### CODE REVIEW

**Comment by znicholls** [comment]
@mikapfl this might help too?

File: tests/unit/test_units.py [hunk 1]
```diff
@@ -170,12 +170,31 @@ def test_gwp_mixture(metric_names, mixture, constituents):
             for fraction, species in constituents:
                 constituent_sum_gwp += (fraction * unit_registry(species)).to("CO2")
             mixture_gwp = unit_registry(mixture).to("CO2")
-            # rounding might be happening, therefore atol=0.6
+
             np.testing.assert_allclose(
-                constituent_sum_gwp.magnitude, mixture_gwp.magnitude, atol=0.6
+                constituent_sum_gwp.magnitude,
+                mixture_gwp.magnitude,
+                rtol=1e-3,  # rounding of reported metric
```
Comment (znicholls) [inline tests/unit/test_units.py:177]: could also undo this change if you want

**Comment by mikapfl** [comment]
The conversions that are defined here are the first non-conservative conversion, are they? As an example, if I do:
```python
a = unit_registry('HFC410A')
with unit_registry.context('HFC410a_conversions'):
    b = a.to('HFC32')
with unit_registry.context('AR4GWP100'):
    np.testing.assert_allclose(a.to('CO2'), b.to('CO2'))
```
the assertion will fail. Without deeper knowledge about HFCs, this is a bit surprising, I think. I also can't really think of a use case right now. Sure, I might like to split HFC410A into its constituents, but the context will not really help me with it because I have no programmatic way to find out which are these constituents. If I had a function to get the constituents, I could see the context be useful like this:
```python
a = unit_registry('HFC410A')
b = []
with unit_registry.context('HFC410a_conversions'):
    for c in get_constituents('HFC410a'):
        b.append(a.to(c))
```

File: tests/unit/test_units.py (hunk 1 above)
Comment (mikapfl) [reply tests/unit/test_units.py:177]: I'm fine either way.

**Comment by znicholls** [comment]
> If I had a function to get the constituents, I could see the context be useful like this

Oh nice that is indeed much better. Maybe then we should add a function like `split_into_constituents` (or whatever name makes sense to you) that uses the information in `data/mixtures.json` to do the splitting? Feel free to close this MR and simply include whatever functionality you think is useful in #11 (or a follow up PR to #11).
